MultiplayerMod/Game/Objects/KPrefabIdEvents.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using HarmonyLib;
+
+namespace MultiplayerMod.Game.Objects;
+
+[HarmonyPatch(typeof(KPrefabID))]
+public static class KPrefabIdEvents {
```

Comment:
Would it make sense to merge it with KInstantiateEvents ? To have KPrefabIdEvents.Created and KPrefabIdEvents.Deseririalize ?

Reply:
Nope, tried that before, these are different things.